nvim-client-0.1.0-1.rockspec

```diff
@@ -1,8 +1,8 @@
 package = 'nvim-client'
 version = '0.1.0-1'
 source = {
-  url = 'https://github.com/neovim/lua-client/archive/' .. version .. '.tar.gz',
-  dir = 'lua-client-' .. version,
+  url = 'git://github.com/phodge/lua-client.git',
```

Comment:
Are you sure this should not be the `.tar.gz` form? I don't know much about luarocks , I'm just basing this on the old code.

Reply:
I found this form on one of the luarocks wiki pages, but it'll have to get reverted before I merge my branch anyway.